src/ssl.c

```diff
@@ -34608,6 +34669,16 @@ int wolfSSL_PEM_write_bio_PUBKEY(WOLFSSL_BIO* bio, WOLFSSL_EVP_PKEY* key)
     return WOLFSSL_SUCCESS;
 }
```

Comment:
something seems wrong with the indentation or braces.  I can't figure out the problem thought.

Reply:
I think its an issue in Github's diff viewer. Its correct if you checkout this branch and look locally.

Reply:
Are you using tabs?

Reply:
Nope, eclipse is set to spaces.